docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+| Visualization default         | is_visualization_default | N/A                               |                             | BOOLEAN    |  --  |  false  | Not by data submitter                                                 | Default: false                                                    | Annotation shown in visualization applications by default (e.g. neuroglancer)                                                                                                                                                                                                                                                      |
+
+## 6. Annotation File Metadata
+
+### 6.1 Point
+Point files are newline-delimited json files ([ndjson](https://clue.engineering/2018/introducing-reactphp-ndjson)). Each
+line contains a nested dictionary. 
+
+| Data elements | metadata key 1 | Sub-elements | metadata key 2 | Value type | Unit | Default | Requirement | Controlled vocabularies | Description                                                                      |
+|---------------|----------------|--------------|----------------|------------|:----:|:-------:|-------------|-------------------------|----------------------------------------------------------------------------------|
+| Location      | location       | x            | x              |  FLOAT     | pix  |   --    | MUST        |                         | x coordinate of the point in pixels, origin is the pixel origin of the 3D image. |
+|               |                | y            | y              |  FLOAT     | pix  |   --    | MUST        |                         | x coordinate of the point in pixels, origin is the pixel origin of the 3D image. |
+|               |                | z            | z              |  FLOAT     | pix  |   --    | MUST        |                         | x coordinate of the point in pixels, origin is the pixel origin of the 3D image. |
+
+### 6.2 OrientedPoint
+OrientedPoint files are newline-delimited json files ([ndjson](https://clue.engineering/2018/introducing-reactphp-ndjson)). Each
+line contains a nested dictionary. 
+
+| Data elements | metadata key 1       | Sub-elements | metadata key 2 | Value type            | Unit | Default | Requirement   | Controlled vocabularies | Description                                                                      | 
+|---------------|----------------------|--------------|----------------|-----------------------|:----:|:-------:|---------------|-------------------------|----------------------------------------------------------------------------------|
```

Comment:
Is lacking the 'type' field that is actually present in the files.

Reply:
Can we add that to this spec?